director/scripts/test/JSON_Visualization_README.txt

```diff
@@ -0,0 +1,40 @@
+This documentation explains the different parts of the JSON file needed for the
+visualization of data during a simulation:
+
+"model_file": This is the directory containing the file for the robot/plant
+              description (like an urdf file).
+
+"weld-body": This is whether the user wants "WeldFrames()" to be called on the
+             robot/plant. If this property does not exist then the "WeldFrames()"
+             function will not be called
+
+"data": This will be a list of all the different visualizations/massages that
+        the user wants to be displayed. The 3 types of visualizations will be
```

Comment:
Unclear. 3 types of visualizations...but "so far there are 2 types of visualization,both of which are kinematic data." What does this mean?

Per our earlier discussions, separate out the "what (data, com, lcm)" from the "how" (line, point)